### PR TITLE
feat: global theme update

### PR DESCRIPTION
**Pull request by paulgrym**
## Jira issue

JIRA issue: [FSU-40](https://klau.atlassian.net/browse/FSU-40)

## Describe your changes

I've added font from google fonts to Storybook and in Frontend directory.

I've changed 'theme' according to Figma, but I'm not sure whether it should looks like that. Please let me know what would be the best approach to 'theme' object. 

## Checklist before requesting a review

- [ ] Checked which branch you would merge.
- [ ] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [ ] Performed a self-review of code (bugs, clean code rules, etc).
- [ ] Checked if added variables have unique names.
- [ ] Checked if added code/function is not a duplicate. 
- [ ] Tests have been added if possible.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Comment by im-klau** [comment]
love it! well done! 🙌 

I just need to pull this branch and test it locally to make sure everything works :D

File: packages/design-system/components/theme.js [hunk 1]
```diff
@@ -1,23 +1,162 @@
...
-	spacings: {
-		text: 24,
-		image: 64,
-	},
+  containerHeight: "100vh",
+  defaultPadding: "16px",
+  fontStyles: {
+    primaryFontFamily: "'Poppins', sans-serif",
+  },
+  colors: {
```
Comment (im-klau) [inline packages/design-system/components/theme.js:7]: I'd say we need just 3 different objects containing our color values: `primary`, `secondary`, and `neutrals`. These 3 should be enough for our project.

**Comment by im-klau** [comment]
@paulgrym let me know if you need any help with resolving conflicts after updating the global theme according to new colors:)

**Review by nconnect1789 — APPROVED** [review]
Looks great 🚀

File: packages/design-system/components/theme.js [hunk 2]
```diff
@@ -1,23 +1,162 @@
...
+    primary: {
+      100: "#FCDDDF",
```
Comment (im-klau) [inline packages/design-system/components/theme.js:9]: looks like the shades are mixed up 🤭

according to the design the `primary[100]` should be `#C1121F` which is `primary[500]` here :D

**Pull request merged**